src/Umbraco.Infrastructure/Composing/CompositionExtensions/Repositories.cs

```diff
@@ -1,7 +1,9 @@
-﻿using Umbraco.Core.Persistence.Repositories;
+﻿using Umbraco.Core;
+using Umbraco.Core.Composing;
+using Umbraco.Core.Persistence.Repositories;
 using Umbraco.Core.Persistence.Repositories.Implement;
 
-namespace Umbraco.Core.Composing.CompositionExtensions
+namespace Umbraco.Web.Composing.CompositionExtensions
```

Comment:
This namespace change seems odd, this has to do with Repositories which are most definitely at the core level